sdcm/cluster_k8s/__init__.py

```diff
@@ -320,9 +320,6 @@ def image(self) -> str:
     def ipv6_ip_address(self):
         raise NotImplementedError()
 
-    def restart(self):
-        raise NotImplementedError("Not implemented yet")  # TODO: implement this method.
-
```

Comment:
How this removal related to caching IP addresses?

Reply:
removed